bld/unit_testers/build-namelist_test.pl

```diff
@@ -1673,7 +1690,7 @@ sub cat_and_create_namelistinfile {
   &cleanup();
   # Run FATES mode for several resolutions and configurations
   my $clmoptions = "-bgc fates -envxml_dir . -no-megan";
-  my @clmres = ( "1x1_brazil", "5x5_amazon", "4x5", "1.9x2.5" );
+  my @clmres = ( "4x5", "1.9x2.5" );
```

Comment:
Explaining the mods in this file:
- 1x1_brazil and 5x5_amazon are not among the 16-pft files